### ISSUE TITLE
Drag and drop cells between notebooks

### ISSUE THREAD

**Issue by simonbyrne** [jupyter/notebook#2524]
The ability to copy-paste cells between notebooks is very much appreciated, thank you! Any chance we could have drag-and-drop as well?

**Comment by takluyver** [comment]
It would definitely make sense, but I don't know much about how to implement drag-and-drop. Are you familiar with javascript?

**Comment by simonbyrne** [comment]
Not really, sorry (otherwise I would have tried to help out). There is a guide here: https://www.html5rocks.com/en/tutorials/dnd/basics/

**Comment by crstnbr** [comment]
This would be a great feature!
